_includes/dev-docs/native-image-asset-sizes.md

```diff
@@ -0,0 +1,31 @@
+There are two methods for defining sizes for image-like assets (`image` and `icon`).  Both are shown below, but the first example (using `sizes`) is more widely supported by demand partners.
+
+1. Using `mediaTypes.native.image.sizes` (or `mediaTypes.native.icon.sizes` for icons):
+
+    ```javascript
```

Comment:
When I run this locally, I'm seeing this appear as in-line code as opposed to a code block with syntax highlighting.

Reply:
Should be fixed by a921d39